Core/HLE/sceAtrac.cpp

```diff
@@ -1629,6 +1629,9 @@ static u32 sceAtracGetNextSample(int atracID, u32 outNAddr) {
 			}
 			if (numSamples > atrac->SamplesPerFrame())
 				numSamples = atrac->SamplesPerFrame();
+			if (atrac->bufferState_ == ATRAC_STATUS_STREAMED_LOOP_FROM_END && numSamples + atrac->currentSample_ > atrac->endSample_) {
+				atrac->bufferState_ = ATRAC_STATUS_ALL_DATA_LOADED;
```

Comment:
This doesn't really make sense.

The "loop from end" state basically means that you have a 200 KB music file but have a buffer that is 100 KB.  So it can obviously never fully fit inside that buffer.  `ATRAC_STATUS_ALL_DATA_LOADED` means the entire music file has been fit inside the provided buffer.

What led you to believe it should transition the state here?

-[Unknown]

Reply:
It just repeat that play the buffer.it would call sceAtracReleaseAtracID when play another Atrac